File: Aug14/Troy/Q4.java
class Solution {
    public String reverseString(String s) {
        int start = 0;
        int end = s.length() - 1;
        StringBuilder result = new StringBuilder();
        result.append(s);
        while(start < end) {
            char temp = result.charAt(start);
            result.setCharAt(start, result.charAt(end));
            result.setCharAt(end, temp);
            start++;
            end--;
        }
        return result.toString();
    }
}

File: Aug14/Carl/Q4.py
class Solution:
    def reverseString(self, s):
        """
        :type s: str
        :rtype: str
        """
        toRet = ""
        for i in range(0, len(s)):
            toRet = s[i]+toRet
        return toRet

S = Solution()
print(S.reverseString('1'))